ghcide/src/Development/IDE/GHC/Compat.hs

```diff
@@ -449,7 +449,7 @@ setUpTypedHoles df
   $ df
   { refLevelHoleFits = Just 1   -- becomes slow at higher levels
```

Comment:
So, I think these things are also user-configurable. Generally it's nice if we don't overwrite user settings if they've set them. So can we adjust these so that they use the value from `df` if it's not `Nothing` and otherwise use these values? I actually think that's what the old PR did?

Reply:
Now respecting previous values like the other PR.